tests/test_other.py

```diff
@@ -9594,17 +9594,17 @@ def test_minimal_runtime_code_size(self):
 
     if self.is_wasm_backend():
       test_cases = [
-        (opts, hello_world_sources, {'a.html': 1445, 'a.js': 484, 'a.wasm': 176}),
-        (opts, hello_webgl_sources, {'a.html': 1565, 'a.js': 4663, 'a.wasm': 11809}),
-        (opts, hello_webgl2_sources, {'a.html': 1565, 'a.js': 5172, 'a.wasm': 11809}) # Compare how WebGL2 sizes stack up with WebGL 1
+        (opts, hello_world_sources, {'a.html': 1205, 'a.js': 484, 'a.wasm': 176}),
+        (opts, hello_webgl_sources, {'a.html': 1335, 'a.js': 4663, 'a.wasm': 11809}),
+        (opts, hello_webgl2_sources, {'a.html': 1335, 'a.js': 5172, 'a.wasm': 11809}) # Compare how WebGL2 sizes stack up with WebGL 1
```

Comment:
what's the reason for this improvement?

Reply:
With dynamic loader generation, the `binary()` and `script()` functions are only included on demand. Although the .html sizes contain unminified JS until #9990 lands, so these numbers are not very meaningful until that.